java/cuvs-java/src/main/java22/com/nvidia/cuvs/internal/BruteForceIndexImpl.java

```diff
@@ -154,44 +155,38 @@ private IndexReference build() throws Throwable {
       MemorySegment datasetMemSegment = dataset != null? ((DatasetImpl) dataset).seg:
           Util.buildMemorySegment(resources.getArena(), vectors);
 
-      long cuvsResources = resources.getMemorySegment().get(cuvsResources_t, 0);
-      MemorySegment stream = arena.allocate(cudaStream_t);
-      var returnValue = cuvsStreamGet(cuvsResources, stream);
```

Comment:
I don't think I understood what the old code is doing here.

I see that we call `cuvsStreamGet()`, which presumably populates the `stream` `MemorySegment` with the appropriate CUDA-stream.  I don't see how/where said `stream` was subsequently used.

It looks like @ldematte's new code just does away with the stream entirely.  Was this never required in the first place?

Reply:
I think that could have been a "copy-paste" error, or some leftover.
We do a `cuvsStreamSync` at the end of the build, which should automatically sync the streams associated with the resource.